tutorials/javaFxPart1.md

```diff
@@ -109,6 +109,30 @@ public class Duke extends Application {
 }
 ```
 
+However, if you do already have an existing constructor which takes arguments, you can proceed 
+by creating an overloaded constructor with no arguments, passing the arguments to the existing 
+constructor e.g.,
```

Comment:
In markdown files, we don't use line breaks to maintain a max line length. 
```suggestion
However, if you do already have an existing constructor which takes arguments, you can proceed by creating an overloaded constructor with no arguments, passing the arguments to the existing constructor e.g.,
```

Reply:
Reason: if you do, subsequent minor edits to the text can require multiple lines to be adjusted, to maintain the same width. That is, it increases the 'blast radius' for future minor edits.